recipes/nifti_clib/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "nifti_clib" %}
+{% set version = "2.0.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/NIFTI-Imaging/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 62655d8457bd04ffd80cf2f07fe8c6aac2b8b00a69fbe71f399e3d9623128630
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - expat
+    - zlib
+
+test:
+    commands:
+
+about:
+    home: https://github.com/NIFTI-Imaging/nifti_clib
+    license: Public Domain
+    license_file: LICENSE
+    summary: 'C library for IO with the nifti file format.'
+
+extra:
+    recipe-maintainers:
+        - leej3
```

Comment:
Please check with `leej3` that he is ok to be listed as a maintainer here. You should add yourself here as well.

Reply:
@leej3, please confirm